Buildings/Experimental/DHC/Networks/BaseClasses/DifferenceEnthalpyFlowRate.mo

```diff
@@ -0,0 +1,196 @@
+  Fluid.Sensors.EnthalpyFlowRate senEntFlo2(
+    redeclare final package Medium=Medium,
+    final m_flow_nominal=m_flow_nominal,
+    final tau=tau,
+    final initType=initType,
+    final allowFlowReversal=allowFlowReversal,
+    final h_out_start=h_out_start)
+    "Enthalpy flow rate of fluid stream 2"
+    annotation (Placement(transformation(extent={{10,-70},{-10,-50}})));
+  Buildings.Controls.OBC.CDL.Continuous.Add dif(final k1=1, final k2=-1)
+    "Compute the difference"
+    annotation (Placement(transformation(extent={{40,10},{60,30}})));
+  Modelica.Blocks.Continuous.Integrator int(y(unit="J")) if have_integrator
+    "Time integral computation"
+    annotation (Placement(transformation(extent={{40,-30},{60,-10}})));
+equation
+  connect(port_a1,senEntFlo1.port_a)
+    annotation (Line(points={{-100,60},{-10,60}},color={0,127,255}));
+  connect(senEntFlo1.port_b,port_b1)
+    annotation (Line(points={{10,60},{100,60}},color={0,127,255}));
+  connect(port_b2,senEntFlo2.port_b)
+    annotation (Line(points={{-100,-60},{-10,-60}},color={0,127,255}));
+  connect(senEntFlo2.port_a,port_a2)
+    annotation (Line(points={{10,-60},{100,-60}},color={0,127,255}));
+  connect(dif.y, dH_flow)
+    annotation (Line(points={{62,20},{120,20}}, color={0,0,127}));
+  connect(senEntFlo1.H_flow, dif.u1)
+    annotation (Line(points={{0,49},{0,26},{38,26}}, color={0,0,127}));
+  connect(senEntFlo2.H_flow, dif.u2)
+    annotation (Line(points={{0,-49},{0,14},{38,14}}, color={0,0,127}));
+  connect(int.y, E)
+    annotation (Line(points={{61,-20},{120,-20}}, color={0,0,127}));
+  connect(dif.y, int.u) annotation (Line(points={{62,20},{80,20},{80,0},{20,0},{
+          20,-20},{38,-20}}, color={0,0,127}));
+  annotation (
+    defaultComponentName="senDifEntFlo",
+    Icon(
+      graphics={
+        Ellipse(
+          fillColor={245,245,245},
+          fillPattern=FillPattern.Solid,
+          extent={{-70,-70},{70,70}}),
+        Line(
+          points={{0,70},{0,40}}),
+        Line(
+          points={{22.9,32.8},{40.2,57.3}}),
+        Line(
+          points={{-22.9,32.8},{-40.2,57.3}}),
+        Line(
+          points={{37.6,13.7},{65.8,23.9}}),
+        Line(
+          points={{-37.6,13.7},{-65.8,23.9}}),
+        Ellipse(
+          lineColor={64,64,64},
+          fillColor={255,255,255},
+          extent={{-12,-12},{12,12}}),
+        Polygon(
+          rotation=-17.5,
+          fillColor={64,64,64},
+          pattern=LinePattern.None,
+          fillPattern=FillPattern.Solid,
+          points={{-5.0,0.0},{-2.0,60.0},{0.0,65.0},{2.0,60.0},{5.0,0.0}},
+          origin={0,0}),
+        Ellipse(
+          fillColor={64,64,64},
+          pattern=LinePattern.None,
+          fillPattern=FillPattern.Solid,
+          extent={{-7,-7},{7,7}}),
+        Line(
+          points={{100,0},{70,0}},
+          color={0,0,127}),
+        Text(
+          extent={{173,26},{79,-4}},
+          lineColor={0,0,0},
+          textString="ΔH_flow"),
```

Comment:
`"ΔH_flow"` will cause token recognition error when parsing it with ANTLR, 
<img width="596" alt="image" src="https://user-images.githubusercontent.com/23491921/99441614-fbed3b80-28cc-11eb-98ef-6cb93f23ab98.png">

Reply:
Addressed.